application/decoder/CdlLogLine.py

```
@@ -0,0 +1,84 @@
+        self.type = LINE_TYPE["EXECUTION"]
+        self.logTypeId = int(line)
+
+
+    def parseIfJson(self, value):
+        '''
+            Try parsing the variable as a json object.
+            If it was unsuccessful, then return the value.
+        '''
+        try:
+            return json.loads(value)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)